batch/batch-runner.sh

```
@@ -298,6 +302,78 @@ get_retries() {
+    raw="$(printf '%s' "$raw" | tr '[:upper:]' '[:lower:]')"
+  fi
+
+  case "$raw" in
+    economy|standard|premium)
+      printf '%s\n' "$raw"
+      ;;
+    "")
+      printf '%s\n' "standard"
+      ;;
+    *)
+      echo "WARN: Invalid spend_tier \"$raw\" in ${PROFILE_FILE#$PROJECT_DIR/}; falling back to standard." >&2
```

Shellcheck (0.11.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)